tests/SpecTests/client-side-encryption/tests/types.json

```diff
@@ -504,7 +504,7 @@
             }
           },
           "result": {
-            "errorContains": "element of type: double"
+            "errorContains": "Cannot use deterministic encryption for element of type: double"
```

Comment:
This 

Looking at the [loadBalanced-latest Evergreen task log](https://evergreen.mongodb.com/lobster/evergreen/task/mongo_php_library_test_loadBalanced__php_edge_versions~latest_stable_versions~latest_driver_versions~latest_dev_test_loadBalanced_patch_ec68b3cb19afffcdb9685f2daaf1d3a8159eb342_62712a550305b94b0eabf341_22_05_03_13_12_54/0/task#bookmarks=0%2C4283&l=1&shareLine=373), I see that the following server version is downloaded:

```
db version v6.1.0-alpha-326-g2d42bc1
Build Info: {
 "version": "6.1.0-alpha-326-g2d42bc1",
 "gitVersion": "2d42bc1dece6cde630ca308be4914f655ffa8f76",
 "openSSLVersion": "OpenSSL 1.1.0l  10 Sep 2019",
 "modules": [
     "enterprise"
 ],
 "allocator": "tcmalloc",
 "environment": {
     "distmod": "debian92",
     "distarch": "x86_64",
     "target_arch": "x86_64"
 }
}
```

This spec test (among others) [fails the following assertion](https://evergreen.mongodb.com/lobster/evergreen/test/mongo_php_library_test_loadBalanced__php_edge_versions~latest_stable_versions~latest_driver_versions~latest_dev_test_loadBalanced_patch_ec68b3cb19afffcdb9685f2daaf1d3a8159eb342_62712a550305b94b0eabf341_22_05_03_13_12_54/0/88f37c005ea45489a8a008873b89ded2/#bookmarks=0%2C5&l=1&shareLine=1):

```
FAILURE:  (PHPUnit\Framework\ExpectationFailedException)
MongoDB\Tests\SpecTests\ClientSideEncryptionSpecTest::testClientSideEncryption with data set "types: type=double" (stdClass Object (...), array(stdClass Object (...)), array(), array(stdClass Object (...)), stdClass Object (), 'default', 'default')
Failed asserting that 'Bulk write failed due to previous MongoDB\Driver\Exception\RuntimeException: mongocryptd error: Cannot encrypt element of type: double:' contains "cannot use deterministic encryption for element of type: double".
/data/mci/829fa8908c65a0c6198bc88eb634f5bb/src/tests/SpecTests/ErrorExpectation.php:153
/data/mci/829fa8908c65a0c6198bc88eb634f5bb/src/tests/SpecTests/Operation.php:234
/data/mci/829fa8908c65a0c6198bc88eb634f5bb/src/tests/SpecTests/ClientSideEncryptionSpecTest.php:132
```

The only failures are in `types.json`, which I see were never included in the original commit for [DRIVERS-2260](https://jira.mongodb.org/browse/DRIVERS-2260): https://github.com/mongodb/specifications/commit/02b4275d4e53dcadf87682b9e07f08a86a7b81d5

It looks like these changes are an unintended reversion of the commit from [DRIVERS-2272](https://jira.mongodb.org/browse/DRIVERS-2272): https://github.com/mongodb/specifications/commit/6a7158d51b4c41f2f4a9c1293c5e1dceb93ab5c2. Those changes were applied in https://github.com/mongodb/mongo-php-library/commit/452bc0fe66a8b615787ac477b2514220e6d88434 (#916) for [PHPLIB-839](https://jira.mongodb.org/browse/PHPLIB-839).